docs/attributes-registry/linux.md

```diff
@@ -0,0 +1,22 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Linux
+
+## Linux Memory Attributes
+
+Describes Linux Memory attributes
+
+| Attribute            | Type   | Description            | Examples                       | Stability                                                        |
+| -------------------- | ------ | ---------------------- | ------------------------------ | ---------------------------------------------------------------- |
+| `linux.memory.state` | string | The Linux memory state | `reclaimable`; `unreclaimable` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
I think this attribute is specific to slab memory, correct? Not all memory would have these states, so I'm suggesting to rename to `linux.slab.memory.state`.

Reply:
Sounds good to move it into `linux.slab.memory.state`, but I think there could be another `linux` memory metric that could reuse the `reclaimable` state. See `KReclaimable` in https://man7.org/linux/man-pages/man5/proc_meminfo.5.html (Kernel allocations that the kernel will attempt to reclaim under memory pressure.). Given this future use case, should we keep it `linux` generic or it will make sense to create a `linux.kernel.memory.state` and duplicate the attribute?

Reply:
Ah, thanks for the explanation!

It seems the attribute will have different values depending on the metric, which seems to be confusing:
- slab memory would have values mapped to `sreclaimable` and `sunreclaim`
- kernel would only have one value - `kreclaimable`. Perhaps kernel memory metric would not even have such attribute and would be called something like `system.linux.kernel.reclaimable.memory` 

So on the first sight there won't be any duplication and `linux.slab.memory.state` makes the most sense for now.

Reply:
agree, `linux.memory.slab.state` looks good.

as for groups: the group in yaml is implementation detail + the thing being rendered as an individual table. I think rendering all `linux.memory` attributes in one table makes sense. Attribute definitions and grouping is irrelevant to how these attributes are used on metrics.

We still render metric-specific attributes here https://github.com/open-telemetry/semantic-conventions/blob/48b89870580518b175b924e36284678c263e7c22/docs/system/system-metrics.md#metric-systemlinuxmemoryslabusage, so it all looks good to me.

Reply:
Makes sense, `slab` namespace included in the attribute: https://github.com/open-telemetry/semantic-conventions/pull/1078/commits/58fad8529029cfbf2e8a47e442179895e128cd8b

Thanks pointing this out.